### ISSUE TITLE
How do I validate on my own validation set?

### ISSUE THREAD

**Issue by Wenlin-Chen** [luckycallor/InsightFace-tensorflow#16]
Hi! I was wondering if I could validate the model on my own validation set, because I'm testing my face detector. I have cropped the faces on LFW data set and saved them as images. Can I feed those into the ArcFace model to do inference? Thanks!
